### PR TITLE
add include pre-processing example

### LINKED ISSUE #21: FR: Macro that "shows" included code
It would be cool if there was a macro that expanded the code included from 
`include(file.jl)` into showing the full source code for this file when building / translating the files to proper format. 

The way _I personally_ would use it is to have the source in a single file, everything there well defined and clear. This would make it even easier to use source files used for tests in the format that Literate.jl uses.

Initially I thought of posting this to Documenter.jl but I wasn't sure. So I am tagging @mortenpi  as well!

### PR DESCRIPTION
**Pull request by Datseris**
Thanks a lot for Literate.jl it is super helpful!!!

Closes #21

### CODE REVIEW

**Comment by Datseris** [comment]
ping

File: docs/src/customprocessing.md [hunk 1]
```diff
@@ -45,3 +46,58 @@ now simply give this function to the generator, for example:
...
+## Example: Replacing `include` calls with included code
+Let's say that we have some individual example files `file1, file2, ...` etc.
+that are _runnable_ and also following the style of Literate. These files could be for example used in the test suite of your package.
+
+We want to group them all into a single page in our documentation, but we
+do not want to copy paste the content of `file1, ...` for robustness: the files are included in the test suite and some changes may occur to them. We want these changes to also be reflected in the documentation.
+
+A very easy way to do this is using `preprocess` to interchange `include` statements with file content. First, create a runnable `.jl` following the format of Literate (the following example comes from the documentation of the Julia package [`TimeseriesPrediction`](https://github.com/JuliaDynamics/TimeseriesPrediction.jl), which uses this approach to create some pages)
+```julia
+# # Spatio-Temporal Prediction Examples
```
Comment (fredrikekre) [inline docs/src/customprocessing.md]: While I appreciate this real-world example, perhaps it is better with something simpler, for example
```
# # Replace includes
# This is an example to replace `include` calls with the actual file content.

include("file1.jl")
include("file2.jl")
```
I think that the message might be clearer that way? It is still good to link to your package of course, as an example.

**Comment by Datseris** [comment]
No problem! I did the requested changes, let me know if there is something more!

**Comment by fredrikekre** [comment]
Thanks!

**Pull request merged**